cluster-autoscaler/FAQ.md

```diff
@@ -1124,20 +1124,19 @@ Assumption: We assume that the developer executing the below stages wants to syn
 
 #### Stage C:  Create Sync Branch, Perform Merge, Fix Vendor
 1. Create a sync branch: `git checkout -b sync-upstream-v1.x.0`. For example, if you are syncing against `1.26.0`, this would be: `git checkout -b sync-upstream-v1.26.0`
-1. Merge against the `releaseBranch` (which you had hard-reset to the `releaseCommitId` earlier): `git merge upstream-release-1.2x`
+1. Merge against the `releaseBranch` (which you had hard-reset to the `releaseCommitId` earlier): `git merge upstream-release-1.x.0`
    - This is where changes of master get merged in. advantage of merging is it won’t change the commit hashes of already existing commits.
    - Accept all vpa changes.
    - Accept our changes in `go.mod` and `go.sum` and `vendor` directory.  (The `hack/update-vendor.sh` script which will be executed in later step expected to fix things)
 1. In `cluster-autoscaler/go.mod`, upgrade versions of `machine-controller-manager-provider-aws`,  `machine-controller-manager-provider-azure` to the latest available release.
-1. Run update vendor script after changing to `cluster-autoscaler` directory:  `./hack/update-vendor.sh 1.x.y` 
+1. Run update vendor script after changing to `cluster-autoscaler` directory:  `./hack/update-vendor.sh 1.x.0` 
    - If the above still gives test issues then use `rsync` or `diff -rq` to figure out differences in `vendor` directory between upstream and our fork and synchronize them manually.
-1. Create a new file `cluster-autoscaler/SYNC-CHANGES/SYNC_CHANGES-1.x.y.md` summarily describing the changes done. Follow existing convention for sync changes. Use upstream release notes as a guide when needed.
+1. Create a new file `cluster-autoscaler/SYNC-CHANGES/SYNC_CHANGES-1.x.0.md` summarily describing the changes done. Follow existing convention for sync changes. Use upstream release notes as a guide when needed.
```

Comment:
kindly commit this directly. If there are many such suggestions you could also add them to batch and then commit them from the UI itself.
```suggestion
1. Create a new file `cluster-autoscaler/SYNC-CHANGES/SYNC_CHANGES-1.x.md` summarily describing the changes done. Follow existing convention for sync changes. Use upstream release notes as a guide when needed.
```

Reply:
Hi Himanshu, this suggestion is outdated as the change was already made. ie file name is already  `cluster-autoscaler/SYNC-CHANGES/`